### PR TITLE
add adapter for oooo.money

### PR DESCRIPTION
**Pull request by roger-new**
support for bridge.oooo.money

### CODE REVIEW

File: src/adapters/oooo/index.ts [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+            const nativeEvents = await Promise.all([
+                ...bridgeAddress.map(async (address: string, i: number) => {
+                    await wait(300 * i); // for etherscan
+                    let txs: any[] = [];
+                    if (chain === "merlin") {
+                        txs = await getTxsBlockRangeMerlinScan(address, fromBlock, toBlock, {
+                            includeSignatures: ["0x"],
+                        });
+                    } else {
+                        txs = await getTxsBlockRangeEtherscan(chain, address, fromBlock, toBlock, {
```
Comment (vrtnd) [inline src/adapters/oooo/index.ts:53]: need to add explorer endpoints for bitlayer and bsquared

Comment (roger-new) [reply src/adapters/oooo/index.ts:53]: Thanks @vrtnd. Just solve it and fix the merge conflicts. Please help to review. Thanks again.

**Comment by roger-new** [comment]
Hi @vrtnd, could you please help to review? Thanks.

**Comment by roger-new** [comment]
@vrtnd could you please kindly help to review my PR? thanks.

**Comment by vrtnd** [comment]
@roger-new pls fix error 
```
TypeError: data.includes is not a function
    at getBlockTXbyAddress (/bridges-server/src/helpers/btr.ts:51:17)
 ```

**Comment by roger-new** [comment]
> @roger-new pls fix error
> 
> ```
> TypeError: data.includes is not a function
>     at getBlockTXbyAddress (/bridges-server/src/helpers/btr.ts:51:17)
> ```
> 
> added error handling for npm run test, pls pull master branch

Thanks @vrtnd for your comments. I just fixed the issue and no error encountered during testing `npm run test oooo 10000`. What's more, the master branch had been pulled as you indicated. Thanks again.

**Pull request merged**